rdflib/store.py

```diff
@@ -238,8 +241,15 @@ def addN(self, quads: Iterable[Tuple["Node", "Node", "Node", "Graph"]]):
             )
             self.add((s, p, o), c)
 
-    def remove(self, triple, context=None):
-        """Remove the set of triples matching the pattern from the store"""
+    def remove(
+        self,
+        triple: Tuple["Node", "Node", "Node"],
+        context: Optional[Union["Graph", "Identifier"]],
```

Comment:
Why is Graph in this annotation? Isn't the point of this PR to make it always an Identifier?

Reply:
Looks like a good point...